src/Extensions/Logging/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+using Microsoft.Omex.Extensions.Logging.Replayable;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	/// <summary>
+	/// Extension methods for the <see cref="IServiceCollection"/> class
+	/// </summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Add IServiceContext to ServiceCollection
+		/// </summary>
+		public static IServiceCollection AddOmexServiceContext<TServiceContext>(this IServiceCollection serviceCollection)
```

Comment:
Missing `<param>` and `<returns>` comments

Reply:
https://github.com/microsoft/Omex/pull/153#discussion_r391957268